src/pages/index.js

```diff
@@ -68,10 +107,62 @@ editProfileButton.addEventListener("click", () => {
   editProfilePopup.open();
 });
 
+const handleDeleteClick = (card) => {
+  confirmationPopup.open();
+
+  confirmationPopup.setSubmitCallback(() => {
+    confirmationPopup.setButtonText(true, "Deleting...");
+
+    api
+      .deleteCard(card.cardId)
+      .then(() => {
+        card.delete();
+        confirmationPopup.close();
+      })
+      .catch((err) => {
+        console.error(err);
+      })
+      .finally(() => {
+        // set the text content back to whatever it started at
+        confirmationPopup.setButtonText(false);
+      });
+  });
+};
+
+const handleLikeClick = (card) => {
+  if (card.isLiked) {
+    api
+      .unlikeCard(card.cardId)
+      .then(() => {
+        card._handleLikeIcon(false);
```

Comment:
`_handleLikeIcon`

Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.